server/routerlicious/packages/services-telemetry/src/test/lumber.spec.ts

```diff
@@ -203,4 +203,24 @@ describe("Lumber", () => {
 		assert.strictEqual(handleErrorStub.calledOnce, true);
 		assert.strictEqual(engineEmitStub.calledOnce, true);
 	});
+
+	it("Makes sure transform is called if formatters are provided.", () => {
```

Comment:
One suggestion (please ignore if this is out of scope): we could also try unite testing the actual `SanitizationLumberFormatter` implementation. That is, we could have two test cases: one with a Lumber object that does not have sensitive information. We would assert that the object was not modified; in other words, that the `exception` property is the same as before. The other test case would have a Lumber object that includes sensitive information. When that happens, we would expect the formatter to redact that part of the error/exception. We would assert that happened.

Reply:
Can't believe I missed this 😅 definitely within my plans and part of the scope. Added!